backend/src/mongo/connection/index.js

```diff
@@ -1,7 +1,5 @@
 const mongoose =  require('mongoose');
-const dotenv = require("dotenv").config()
```

Comment:
Moved this to `index.js`, check [this comment](https://github.com/nds-fsd/Abd-ha-todo-app/pull/2/files#r1501687470)